packages/lab/src/Flow/types/flow.ts

```diff
@@ -95,7 +98,18 @@ export interface HvFlowNodeSelectParam extends HvFlowNodeSharedParam {
   options?: string[];
 }
 
-export type HvFlowNodeParam = HvFlowNodeSelectParam | HvFlowNodeTextParam;
+export interface HvFlowNodeSliderParam extends HvFlowNodeSharedParam {
+  type: "slider";
+  minPointValue?: HvSliderProps["minPointValue"];
+  maxPointValue?: HvSliderProps["maxPointValue"];
+  markStep?: HvSliderProps["markStep"];
+  divisionQuantity?: HvSliderProps["divisionQuantity"];
+}
```

Comment:
Why stop at these 4 props? What if the user needs (for instance, `formatMark`)? We're already forwarding any props to the `HvSlider`, I think we should allow in on the types too
(we might need to Omit some props)

```suggestion
export interface HvFlowNodeSliderParam extends HvFlowNodeSharedParam,
  Omit<HvSliderProps, keyof HvFlowNodeSharedParam>{
  type: "slider";
}
```